icu4c/source/common/uresbund.cpp

```diff
@@ -47,7 +47,7 @@ TODO: This cache should probably be removed when the deprecated code is
       completely removed.
 */
 static UHashtable *cache = NULL;
-static icu::UInitOnce gCacheInitOnce;
+static icu::UInitOnce gCacheInitOnce U_INITONCE_INITIALIZER;
```

Comment:
missing an '=' ? how does this compile as is?

Reply:
The U_INITONCE_INITIALIZER is defined as { stuff }, which is allowed C++11 syntax, 
`Type variable {initializer};`

Fixed, anyhow.